snake-ladder-ui/src/Components/GameConfig/GameConfig.js

```diff
@@ -0,0 +1,149 @@
+            id="boardColumns"
+            value={boardColumns}
+            onChange={(event) => setBoardColumns(parseInt(event.target.value))}
+          />
+        </div>
+        <div className="mb-3">
+          <label htmlFor="diceCount" className="form-label">Dice Count:</label>
+          <input
+            type="number"
+            className="form-control"
+            id="diceCount"
+            value={diceCount}
+            onChange={(event) => setDiceCount(parseInt(event.target.value))}
+          />
+        </div>
+        <div className="mb-3">
+          <label htmlFor="playerCount" className="form-label">Player Count:</label>
+          <input
+            type="number"
+            className="form-control"
+            id="playerCount"
+            value={playerCount}
+            onChange={(event) => setPlayerCount(parseInt(event.target.value))}
+          />
+        </div>
+
+        <div className="mb-3">
+          <label className="form-label">Snake :</label>
+          {snake.map((position, index) => (
+            <div key={index} className="d-flex">
+              <input
+                type="number"
+                className="form-control me-2"
+                placeholder="Start Position"
+                value={position.start}
+                onChange={(event) => handlePositionChangeSnake(index, "start", event.target.value)}
+              />
+              <input
+                type="number"
+                className="form-control me-2"
+                placeholder="End Position"
+                value={position.end}
+                onChange={(event) => handlePositionChangeSnake(index, "end", event.target.value)}
+              />
+            </div>
+          ))}
+          
+          <button type="button" className="btn btn-secondary mt-2" onClick={handleAddPositionSnake}>
+            Add
+          </button>
+        </div>
+
+        <div className="mb-3">
+          <label className="form-label">Ladder :</label>
+          {Ladder.map((position, index) => (
+            <div key={index} className="d-flex">
+              <input
+                type="number"
+                className="form-control me-2"
+                placeholder="Start Position"
+                value={position.start}
+                onChange={(event) => handlePositionChangeLadder(index, "start", event.target.value)}
+              />
+              <input
+                type="number"
+                className="form-control me-2"
+                placeholder="End Position"
+                value={position.end}
+                onChange={(event) => handlePositionChangeLadder(index, "end", event.target.value)}
+              />
+            </div>
+          ))}
+          
+          <button type="button" className="btn btn-secondary mt-2" onClick={handleAddPositionLadder}>
+            Add
+          </button>
+        </div>
+
+        <button type="submit" className="btn btn-primary">Start Game</button>
```

Comment:
Instead of Start Game we can change it to create game or something more meaningful.

Reply:
sure